packages/terra-tabs/src/StructuralTabs.scss

```diff
@@ -140,6 +140,7 @@
     .tab-menu:focus.is-active {
       background-color: var(--terra-tabs-structural-active-focus-background-color, #fff);
       background-image: var(--terra-tabs-structural-background-image, linear-gradient(to bottom, #007ee0, #007ee0));
+      color: #000;
```

Comment:
Let's set this to color: var(--terra-tabs-structural-active-focus-color, inherit);. This will inherit the parent elements color which is likely inherited from base. Then we should be good to just have 1 instance of color in this selector.